Anatolie/src/main/java/AreaOfCircle.java

```diff
@@ -0,0 +1,10 @@
+public class AreaOfCircle {
+    public static void main(String[] args) {
+
+        double radius;
+        radius = 3;
```

Comment:
Code would looks better with these statements written inline (ex: double radius = 3;)
the same for area variable

Reply:
Done